rust/src/participant.rs

```diff
@@ -171,7 +173,7 @@ impl Participant {
 
     /// Generate a mask seed and mask a local model (dummy).
     fn mask_model() -> (MaskSeed, MaskObject) {
-        let model = Model::from_primitives(vec![0_i32, 1_i32, 2_i32, 3_i32].into_iter()).unwrap();
+        let model = Model::from_primitives(vec![0_f32, 1_f32, 2_f32, 3_f32].into_iter()).unwrap();
```

Comment:
the dummy `model` doesn't fulfill the requirements of the chosen `dummy_config()`, which has `B0` as `BoundType` (line 359). therefore, the model parameters must either be absolutely bounded by `1` or the config `BoundType` must be less constraining like `B2`. otherwise the unmasked model will yield garbage.